test/acceptance/recovery/recovery_test.go

```diff
@@ -21,7 +21,12 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+// TODO-RAFT current tests doesn't force containers to change their IPs
+// we need to add test were the actual container ip changes on stop if possible with testcontainer
```

Comment:
There is no way to do that deterministically. We tried earlier to have a deterministic IP on restart because it was causing issues. Testcontainers maintainers say they don't support the option to have static ip (or force a different one) you can pass the option in the config but it will be ignored by the underlying implem IIRC.

Reply:
then we would terminate and move files around 🤔  or we can do it with helm and find a way to trigger it from here 🤔